### PR TITLE
[RU]MangaLib fix cloudflare errors

### LINKED ISSUE #6601: API Domain setting in LibGroup (RU)
### Source name

MangaLib, SlashLib (YaoiLib), HentaiLib

### Source language

Russian

### Describe your suggested feature

As mentioned in #6069 and [here](https://lib.social/forum/discussion/580104). There are some CF problems. 

To fix it there are multiple API mirrors. I suggest to add option to change API domain in extension settings

### Other details

Example domains (haven't tested  'em all yet):

- api.lib.social
- api2.mangalib.me
- api.imglib.info

### Acknowledgements

- [ ] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/mihonapp/mihon/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by e-shl**
Minimize rateLimit chapter update and removing outdated unnecessary User-Agent
Closes https://github.com/keiyoushi/extensions-source/issues/6601
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by NoPlagiarism** [comment]
Can you also check #6601?

File: lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroup.kt [hunk 1]
```diff
@@ -578,6 +574,10 @@ abstract class LibGroup(
         private const val LANGUAGE_PREF = "MangaLibTitleLanguage"
         private const val LANGUAGE_PREF_TITLE = "Выбор языка на обложке"
 
+        private const val API_DOMAIN_PREF = "REMangaDomain"
```
Comment (Pavkazzz) [inline lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroup.kt]: LibMangaDomain - will be better

**Comment by MikeCruzer** [comment]
Скачал последнее расширение мангалиба с бусти, работает, но через раз выскакивает с этой ошибкой.

User-agent стоит гугловский

App version: 0.15.3 (standard, a9c7cbf, 119, 2024-01-13T19:03Z)
Android version: 13 (SDK 33; build TKQ1.221013.002 test-keys)
Device brand: nubia
Device manufacturer: nubia
Device name: NX679J-EEA (NX679J-EEA)
Device model: NX679J
WebView: Android System WebView 131.0.6778.104

--------- beginning of main
12-16 13:42:30.852  7510  7510 E anade.tachiyomi: Not starting debugger since process cannot load the jdwp agent.
12-16 13:42:30.971  7510  7563 E chromium: [1216/134230.971458:ERROR:variations_seed_loader.cc(39)] Seed missing signature.
12-16 13:42:31.363  7510  7553 E SQLiteLog: (284) automatic index on C(manga_id)
12-16 13:42:31.363  7510  7553 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
12-16 13:43:12.384  8211  8211 E i:error_handler: Not starting debugger since process cannot load the jdwp agent.
12-16 13:43:12.458  8211  8238 E SQLiteLog: (283) recovered 164 frames from WAL file /data/data/eu.kanade.tachiyomi/databases/tachiyomi.db-wal
12-16 13:43:13.449  8479  8479 E anade.tachiyomi: Not starting debugger since process cannot load the jdwp agent.
--------- beginning of system
12-16 13:43:13.473  8479  8479 E ActivityThread: two prestart wait!8479,10254, wait before app application onCreate=true
12-16 13:43:14.570  8479  8512 E SQLiteLog: (5) statement aborts at 2: [PRAGMA journal_mode=TRUNCATE] database is locked
12-16 13:43:14.611  8479  8524 E chromium: [1216/134314.611503:ERROR:variations_seed_loader.cc(39)] Seed missing signature.
12-16 13:43:14.923  8479  8512 E SQLiteLog: (284) automatic index on C(manga_id)
12-16 13:43:14.923  8479  8512 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)

**Comment by e-shl** [comment]
> 0.15.3

Update tachi

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**